client/src/components/Button/Button.js

```diff
@@ -63,10 +47,6 @@ const Button = ({
 Button.propTypes = {
   href: PropTypes.string,
   className: PropTypes.string,
-  icon: PropTypes.oneOfType([
-    PropTypes.string,
-    PropTypes.arrayOf(PropTypes.string),
-  ]),
```

Comment:
I think it would still be valuable for the button to take care of rendering any icons within it, since it's quite common for buttons to have icons.

Reply:
Discussed in person: No change to this PR. This is something for the UI refactoring team to explore further, as the current Button component is isolated to menu use cases, but we'll want a more generic button component for other parts of the UI.